src/main/scala/firrtl/options/phases/WriteOutputAnnotations.scala

```diff
@@ -37,8 +37,12 @@ class WriteOutputAnnotations extends Phase {
 
         filesWritten.get(canonical) match {
           case None =>
-            val w = new FileOutputStream(filename)
-            a.getBytes.foreach(w.write(_))
+            val w = new BufferedOutputStream(new FileOutputStream(filename))
+            a.getBytes match {
+              case arr: mutable.WrappedArray[Byte] => w.write(arr.array)
+              case other => other.foreach(w.write(_))
+            }
+            w.flush()
```

Comment:
Is this flush necessary given that the stream is closed on the next line?

Reply:
I read something on StackOverflow that suggested it was, but further reading indicates it is not, will remove.